src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -396,6 +396,14 @@
       <arg type="s" name="transaction_address" direction="out"/>
     </method>
 
+    <!-- Experimental DownloadPackages -->
+    <method name="DownloadPackages">
+      <!-- array of package names -->
+      <arg type="as" name="queries" direction="in"/>
+      <!-- array of fds -->
+      <arg type="ah" name="fds" direction="out"/>
```

Comment:
To keep things simple, I'm thinking you can drop this argument entirely. We're only returning one kind of fds: fds to RPM files. So all the fds returned by this call will be of the same type. Handle (`h`) args are helpful when you have different kinds of fds being passed and you need to know which index corresponds to what.

Dropping this will make the signature easier to handle on the client side.